ide/gsf.testrunner/nbproject/project.xml

```diff
@@ -79,6 +79,7 @@
                 <friend>org.netbeans.modules.groovy.support</friend>
                 <friend>org.netbeans.modules.gsf.testrunner.ui</friend>
                 <friend>org.netbeans.modules.hudson.ui</friend>
+                <friend>org.netbeans.modules.java.lsp.server</friend>
```

Comment:
I suggest to open up the API to public rather than adding yet another friend.

Reply:
Done.